api/src/main/java/jakarta/json/bind/customization/ParamCustomization.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package jakarta.json.bind.customization;
+
+import java.util.Objects;
+
+import jakarta.json.bind.spi.JsonbProvider;
+
+/**
+ * Decorated parameter customization.
+ */
+public interface ParamCustomization extends JsonbCustomization, ScopelessCustomization {
```

Comment:
`ParamCustomization` as it seems is not really a customization, it is a parameter of `CreatorCustomization`.
Should it follow the same naming pattern, and should it be top level interface?

I'd propose name `CreatorParameter` instead, moving it into CreatorCustomization would be also an option to explore.